src/Krank/Types.hs

```diff
@@ -0,0 +1,11 @@
+module Krank.Types (
+  Violation(..)
+  ) where
+
+import Data.Text (Text)
+
+data Violation = Violation { checker :: Text
+                           , level :: Int
```

Comment:
Can we imagine a `data = Warning | Info | Foo` for `level`?

Reply:
yes. I was actually not happy with this `Int` I drafted once came time to display it.
I'll work on that